src/core/config/remote/enumerations.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the ""License"");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an ""AS IS"" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+/**
+ * Enumerations that represent the component types used in remote config
+ * {@link ComponentsDataWrapper}
+ */
+export enum ComponentType {
+  ConsensusNode = 'consensus nodes',
+  HaProxy = 'ha proxies',
+  EnvoyProxy = 'envoy proxies',
+  MirrorNode = 'mirror nodes',
+  MirrorNodeExplorer = 'mirror node explorers',
+  Relay = 'replays',
```

Comment:
```suggestion
  ConsensusNode = 'consensus-node',
  HaProxy = 'ha-proxy',
  EnvoyProxy = 'envoy-proxy',
  MirrorNode = 'mirror-node',
  MirrorNodeExplorer = 'mirror-node-explorer',
  Relay = 'relay',
```
since this seems to be ending up in the solo-remote-config as yaml, then it should probably be well formed.  Instead of:
```
 apiVersion: v1                                                                                                                                                                                                                           
 data:                                                                                                                                                                                                                                    
   remote-config-data: |                                                                                                                                                                                                                  
     metadata:                                                                                                                                                                                                                            
       name: solo-e2e                                                                                                                                                                                                                     
       lastUpdatedAt: 2024-11-26T22:23:55.286000Z                                                                                                                                                                                         
       lastUpdateBy: jeromy@swirldslabs.com                                                                                                                                                                                               
     version: v0.56.5                                                                                                                                                                                                                     
     clusters:                                                                                                                                                                                                                            
       kind-solo-e2e: solo-e2e                                                                                                                                                                                                            
     components:                                                                                                                                                                                                                          
       replays: {}                                                                                                                                                                                                                        
       ha proxies: {}                                                                                                                                                                                                                     
       mirror nodes: {}                                                                                                                                                                                                                   
       envoy proxies: {}                                                                                                                                                                                                                  
       consensus nodes: {}                                                                                                                                                                                                                
       mirror node explorers: {}                                                                                                                                                                                                          
     commandHistory:                                                                                                                                                                                                                      
       - deployment create                                                                                                                                                                                                                
     lastExecutedCommand: deployment create                                                                                                                                                                                               
 kind: ConfigMap                                                                                                                                                                                                                          
 metadata:                                                                                                                                                                                                                                
   creationTimestamp: "2024-11-26T22:23:55Z"                                                                                                                                                                                              
   labels:                                                                                                                                                                                                                                
     solo.hedera.com/type: remote-config                                                                                                                                                                                                  
   name: solo-remote-config                                                                                                                                                                                                               
   namespace: solo-e2e                                                                                                                                                                                                                    
   resourceVersion: "2842"                                                                                                                                                                                                                
   uid: 1d9fc502-54ed-4d0e-a983-66e1195fc6e1                                                                                                                                                                                              
```